text/lang-detection/model.jl

```diff
@@ -1,13 +1,17 @@
-using Flux
+using Flux       # Main ML library
 using Flux: onehot, onehotbatch, crossentropy, reset!, throttle
 
+using Unicode    # For 'normalize'
+using Random     # for 'shuffle'
+using Statistics # for 'mean'
```

Comment:
Seeing as we are only using one function from these modules, do you think something like this would be better?

```suggestion
using Statistics: mean
```